extensions/ql-vscode/src/config.ts

```diff
@@ -576,6 +623,11 @@ export function getVariantAnalysisDefaultResultsSort(): SortKey {
  */
 const ACTION_BRANCH = new Setting("actionBranch", VARIANT_ANALYSIS_SETTING);
 
+export const VARIANT_ANALYSIS_ENABLE_GHEC_DR = new Setting(
+  "enable_ghec_dr",
```

Comment:
I think it would be better for this name to be consistent with other settings:

```suggestion
  "enableGhecDr",
```